UI/window-basic-main.cpp

```diff
@@ -4292,14 +4292,6 @@ static inline enum video_format GetVideoFormatFromName(const char *name)
 		return VIDEO_FORMAT_I010;
 	else if (astrcmpi(name, "P010") == 0)
 		return VIDEO_FORMAT_P010;
-#if 0 //currently unsupported
-	else if (astrcmpi(name, "YVYU") == 0)
-		return VIDEO_FORMAT_YVYU;
-	else if (astrcmpi(name, "YUY2") == 0)
-		return VIDEO_FORMAT_YUY2;
-	else if (astrcmpi(name, "UYVY") == 0)
-		return VIDEO_FORMAT_UYVY;
-#endif
```

Comment:
I'm not sure if it's worth removing these in this PR as it adds to the scope. These can be removed in a separate follow-up PR.

Reply:
I include this patch in the first version, because the dead code was in the function modified in this PR.
Will remove in the next update